dask/dataframe/io/tests/test_sql.py

```diff
@@ -440,7 +440,7 @@ def test_query_with_meta(db):
     ).select_from(sql.table("test"))
     out = read_sql_query(s1, db, npartitions=2, index_col="number", meta=meta)
     # Don't check dtype for windows https://github.com/dask/dask/issues/8620
-    assert_eq(out, df[["name", "age"]], check_dtype=sys.platform != "win32")
+    assert_eq(out, df[["name", "age"]], check_dtype=sys.int_info.sizeof_digit != 2)
```

Comment:
Should this be:

```suggestion
    assert_eq(out, df[["name", "age"]], check_dtype=sys.int_info.sizeof_digit != 32)
```

Reply:
It definitely should be 2. (I think it's size in bytes.)

On a 32 bit linux system with python 3.9.2 I get:

```bash
python3 -c "import sys; print(sys.int_info)"
sys.int_info(bits_per_digit=15, sizeof_digit=2)
```

on the 32 bit windows version of  python 3.10.6 running under the 32 bit wine (5.0.3) I get:

```bash
wine python.exe -c "import sys; print(sys.int_info)"
sys.int_info(bits_per_digit=15, sizeof_digit=2)
```

And on a 64 bit linux system with python 3.10.6 I get 
```bash
python3 -c "import sys; print(sys.int_info)"
sys.int_info(bits_per_digit=30, sizeof_digit=4)
```